stable/minio/templates/deployment.yaml

```diff
@@ -93,6 +93,11 @@ spec:
                   key: gcs_key.json
             {{- end }}
           livenessProbe:
+            tcpSocket:
```

Comment:
You may also need to add both livenessProbe  and readinessProbe  in templates/statefulset.yaml.

Reply:
will add, thanks!

Reply:
https://github.com/helm/charts/pull/7598